gradle/libs.versions.toml

```diff
@@ -72,4 +79,9 @@ javapoet = { module = "com.squareup:javapoet", version.ref = "javapoet" }
 hilt-android = { module = "com.google.dagger:hilt-android", version.ref = "hilt" }
 hilt-compiler = { module = "com.google.dagger:hilt-compiler", version.ref = "hilt" }
 
-junit = { module = "junit:junit", version.ref = "junit" }
\ No newline at end of file
+junit = { module = "junit:junit", version.ref = "junit" }
+
+compose-material-dialog-core = { module = "io.github.vanpra.compose-material-dialogs:core", version.ref = "dialogCore" }
+compose-material-dialog-datetime = { module = "io.github.vanpra.compose-material-dialogs:datetime", version.ref = "dialogCore" }
+
+serialization-gson = { module = "com.google.code.gson:gson", version.ref = "gson" }
```

Comment:
do we need this library?

Reply:
deleted, forgot to delete.  InitilaIly, I  used Sharedpreferences and saved Profile object using gson